gallery/src/components/NxThreatCounter/NxThreatCounterPage.tsx

```diff
@@ -92,7 +119,14 @@ const NxThreatCounterPage = () =>
                         id="nx-threat-counter-row"
                         liveExample={NxThreatCounterExample}
                         codeExamples={NxThreatCounterCode}>
-      <code className="nx-code">nx-threat-counter</code>s in the default layout.
+      <code className="nx-code">nx-threat-counter</code>s in the default (row) layout.
+    </GalleryExampleTile>
+
+    <GalleryExampleTile title="NxThreatCounter Small Default Example"
+                        id="nx-threat-counter-row"
```

Comment:
Duplicate id here and on the other new ones.  Aside from that generally being invalid, you'll want to use the ids here as you add to the visual tests.

Reply:
> Duplicate id here and on the other new ones. Aside from that generally being invalid, you'll want to use the ids here as you add to the visual tests.

Thanks I missed that, changed the ids and added more visual tests in https://github.com/sonatype/sonatype-react-shared-components/pull/418/commits/8cce8f0e7e32b56afa23137b9a7b1a8cb948a2bd